.gitmodules

```diff
@@ -7,3 +7,7 @@
 	url = https://github.com/tikv/titan.git
 	branch = master
 
+[submodule "librocksdb_sys/rocksdb-cloud"]
+	path = librocksdb_cloud_sys/rocksdb-cloud
+	url = https://github.com/rockset/rocksdb-cloud
```

Comment:
I think we need a fork to keep the build stable. I'll create one under tikv org next week.